### ISSUE TITLE
[Feature request]: Align selected nodes all at once

### ISSUE THREAD

**Issue by louisgreiner** [SchweizerischeBundesbahnen/netzgrafik-editor-frontend#260]
### Preflight Checklist

- [X] I have read the [Contributing Guidelines](https://github.com/SchweizerischeBundesbahnen/netzgrafik-editor-frontend/blob/main/CONTRIBUTING.md) for this project.
- [X] I agree to follow the [Code of Conduct](https://github.com/SchweizerischeBundesbahnen/netzgrafik-editor-frontend/blob/main/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://github.com/orgs/sbb-design-systems/projects/1) for an issue that matches the one I want to file, without success.

### Request type

Request for a new component

### Functionality

Currently, we have to move the nodes and set their position manually. To draw a nice network graphic, one would like to align the nodes (= same x or y position).

We can start think about a feature that would align the selected nodes, or if a user drag a node next to another, it can "stick" to the same x or y position than its neighbor, or any other node close to the same x or y position, as it can be the case for PowerPoint elements

I believe there is already a "stick to near node position" feature, but is there any "stick all selected nodes positions" feature coming ?

**Comment by aiAdrian** [comment]
Indeed, there is a primitive for aligning nodes to the raster. But this method is not yet fully implemented and can be improved.

```typescript
 export const RASTERING_BASIC_GRID_SIZE = 32;

 NodeService.alginNodeToRaster
 NoteService.alginNoteToRaster
```

This problem requires a more detailed specification. It may be worth working with a UX designer. PowerPoint could be a good application to draw inspiration from.

**Comment by aiAdrian** [comment]
First part of featute is complete.
